RussianFilter/sections/specific.txt

```diff
@@ -95,6 +95,7 @@ rmnt.ru##div[id^="ad-in-content-"]
 ideaprog.download##.new_yapr
 ||kod.ru/assets/images/promo/
 starhit.ru##.hb-placeholder
+||aio.media/*/requestAd?appid=
```

Comment:
It is ad networs. Better block `static.aio.media` in ad servers.

Reply:
@stanislav-atr

Reply:
@Alex-302, fixed.